src/mimefactory.rs

```diff
@@ -482,10 +487,15 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
         let grpimage = self.grpimage();
         let force_plaintext = self.should_force_plaintext();
         let skip_autocrypt = self.should_skip_autocrypt();
-        let subject_str = self.subject_str().await;
+        let mut subject_str = self.subject_str().await;
         let e2ee_guaranteed = self.is_e2ee_guaranteed();
         let encrypt_helper = EncryptHelper::new(self.context).await?;
 
+        if subject_str.ends_with("#BCC") {
+            subject_str.truncate(subject_str.len()-4);
+        }
+        info!(self.context, "MimeFactory::render: subject: {}", subject_str);
+        
```

Comment:
I don't think we should change the subject after subject_str() returned. Rather adapt subject_str() (but that's only important if we keep the #BCC thing anyway)

Reply:
Oh yes, good idea. My old code didn't contain the method subject_str(). I've overseen this.
That's the better approach (when possible)

But You need to see groups name with #BCC in chatlist. Only the receipients of the mail shouldn't see it.
Will check.